hack/build-helm-image.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/env bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
```

Comment:
Since these are files authored this year, let's use 2024 instead of 2023

Reply:
ok, done